openedx/core/lib/x_forwarded_for/middleware.py

```diff
@@ -51,6 +51,12 @@ def process_request(self, request):
             # Give some observability into IP chain length and composition. Useful
             # for monitoring in case of unexpected network config changes, etc.
             ip_chain = ip.get_raw_ip_chain(request)
+
+            # .. custom_attribute_name: ip_chain.raw
+            # .. custom_attribute_description: The actual contents of the raw IP chain. Could
+            #      be used to correlate authenticated and unauthenticated requests for the same
+            #      user.
+            set_custom_attribute('ip_chain.raw', ip_chain)
```

Comment:
Do we need to format this list to a string, or is `set_custom_attribute` going to take care of it for us?

Reply:
(Adding a test would be great, and probably quick as well.)

Reply:
1. Added a test.
2. Added formatting.